.github/workflows/ci-testing.yml

```diff
@@ -0,0 +1,93 @@
+name: Cartopy Testing
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python-version: [3.6, 3.8]
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: conda-incubator/setup-miniconda@v2
+      with:
+        activate-environment: test-environment
+        python-version: ${{ matrix.python-version }}
+        channels: conda-forge/label/testing,conda-forge
+
+    - name: Minimum packages
+      # Only run on macos for now
+      # Conda's linux packages don't grab the testing label of matplotlib causing failures due to freetype differences
+      if: ${{ matrix.python-version == '3.6' && matrix.os == 'macos-latest' }}
+      id: minimum-packages
+      run: |
+        echo "PACKAGES=cython=0.28.5 matplotlib=2.2.2 numpy=1.16 owslib=0.17 proj4=5.2.0 scipy=1.2.0" >> $GITHUB_ENV
+        echo "CFLAGS=-stdlib=libc++" >> $GITHUB_ENV
+
+    - name: Latest packages
+      if: ${{ steps.minimum-packages.conclusion == 'skipped' }}
+      run: |
+        echo "PACKAGES=cython fiona matplotlib-base numpy proj pykdtree scipy" >> $GITHUB_ENV
+      
+    - name: Coverage packages
+      id: coverage
+      # only want the coverage to be run on the latest ubuntu
+      if: ${{ matrix.python-version == '3.8' && matrix.os == 'ubuntu-latest' }}
+      run: |
+        echo "PACKAGES=$PACKAGES pytest-cov coveralls" >> $GITHUB_ENV
+        echo "CYTHON_COVERAGE=1" >> $GITHUB_ENV
+        echo "EXTRA_TEST_ARGS=--cov=cartopy -ra" >> $GITHUB_ENV
+
+    - name: Install dependencies
+      run: |
+        PACKAGES="$PACKAGES flufl.lock owslib pep8 pillow pyepsg pyshp pytest"
+        PACKAGES="$PACKAGES pytest-xdist requests setuptools_scm"
+        PACKAGES="$PACKAGES setuptools_scm_git_archive shapely"
+        conda install $PACKAGES
+        conda info -a
+        conda list
+
+    - name: Install Cartopy
+      run: |
+        MPL_CONFIG_DIR=~/.config/matplotlib
+        mkdir -p $MPL_CONFIG_DIR
+        echo "backend" ":" "agg" > $MPL_CONFIG_DIR/matplotlibrc
```

Comment:
This doesn't work?
```suggestion
        echo "backend : agg" > $MPL_CONFIG_DIR/matplotlibrc
```

Reply:
Copy/pasted from before. Not sure why it was that way, good catch.